packages/bot/src/commands/2fa.ts

```diff
@@ -0,0 +1,46 @@
+import { authenticator } from 'otplib';
+import QRCode from 'qrcode';
+import { Context, Telegraf } from 'telegraf';
+import { users } from '../db';
+
+export default function setup2fa(bot: Telegraf<Context>) {
+  bot.command('2fa', async (ctx) => {
+    const service: string = 'FukiPay';
+    const tgUserId: number = ctx.from?.id as any;
+    const secret: string = authenticator.generateSecret();
+    const otpauth: string = authenticator.keyuri((tgUserId as any) as string, service, secret);
+    const qrcode: string = await QRCode.toDataURL(otpauth);
+    const user = await users.findById(tgUserId);
+
+    ctx.replyWithPhoto(
```

Comment:
I'm not sure but, if we don't return, the function may time out, I think you should return